src/tomczyk-kamil/Fsd.Kamil.Client/Fsd.Kamil.Client.Ex3/colorSelector.js

```diff
@@ -0,0 +1,29 @@
+var app = app || {};
+
+(function () {
+    function initialize() {
+        document.querySelectorAll('#color-options').forEach(function (item) {
+            item.addEventListener('click', ColorPickerHandler);
+        });
+    }
+
+    function ColorPickerHandler(event) {
+
+        let colorOptions = $('#color-options');
+        let colorChoice = $('#color-choice');        
+        let selectedItem = $(event.target);
+        let selectedColor = selectedItem.attr('start-color');
+        let newColor = $('#selected-color')
+        let shadow = $('selected');
+        
+        newColor.html(selectedColor);
+
+        colorChoice.removeClass();        
+        colorChoice.addClass(selectedColor);
+        selectedItem.addClass(shadow);
```

Comment:
żeby zrobić cień nie trzeba nadawać klasy i bawić się JS - troche to nadmiarowe. W CSS masz selector :hover co oznacza "najechany element". Wystarczy zrobić .xxx:hover {background: red;} i każdy najechany element z klasą xxx zmieni kolor na czerwony po najechaniu